webservices/common/models/efiling.py

```diff
@@ -0,0 +1,118 @@
+import sqlalchemy as sa
```

Comment:
I put the efiling filings with filings. I think we want to standardize if the efilings are going to be grouped with their processed element or grouped together. For me it was good to have the processed model in the same file so I could compare and check for consistency.

It can go either way, I just want it to be consistent.